pkg/jobrunaggregator/jobruntestcaseanalyzer/analyzer.go

```diff
@@ -86,11 +87,28 @@ func (s *testCaseAnalyzerJobGetter) filterJobsForPayload(allJobs []jobrunaggrega
 			(len(s.infrastructure) != 0 && s.infrastructure != getJobInfrastructure(job.JobName)) {
 			continue
 		}
+
+		if s.isJobNameFiltered(job.JobName) {
+			continue
+		}
+
 		jobs = append(jobs, job)
 	}
 	return jobs
 }
 
+func (s *testCaseAnalyzerJobGetter) isJobNameFiltered(jobName string) bool {
+
+	if s.excludeJobNames != nil {
```

Comment:
minor: kube style would be to invert this and have

```go
if s.excludeJobNames == nil{
  return false
}

for key := range..
```

Also note that you don't need the if block at all for golang, you can iterate over a nil map: https://go.dev/play/p/FStUZvQBOT2